Hmm, failing CI for some reason...

```
103   const predicates = browserPredicates[ua.getBrowser().name] || {};
                                           ~~~~~~~~~~~~~~~~~~~~
src/browser-capabilities.ts(103,40): error TS2538: Type 'undefined' cannot be used as an index type.
```

--- a/src/browser-capabilities.ts
+++ b/src/browser-capabilities.ts
@@ -87,7 +87,8 @@ const browserPredicates: {
   },
   'Firefox': {
     es2015: since(51),
-    push: since(36),
+    // Firefox bug - https://bugzilla.mozilla.org/show_bug.cgi?id=1409570
+    push: () => false,
     serviceworker: since(44),
     modules: () => false,
   },

